includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -69,11 +69,6 @@ private function getLangWithFallback( $lang ) {
 			'help' => 'http://community.wikia.com/wiki/Help:Contents',
 			'media-kit' => 'http://www.wikia.com/mediakit',
 			'media-kit-contact' => null,
-			'social-facebook' => 'https://www.facebook.com/getfandom',
-			'social-twitter' => 'https://twitter.com/getfandom',
-			'social-reddit' => 'https://www.reddit.com/r/wikia',
-			'social-youtube' => 'https://www.youtube.com/channel/UC988qTQImTjO7lUdPfYabgQ',
-			'social-instagram' => 'https://www.instagram.com/getfandom/',
 			'app-store' => 'https://itunes.apple.com/developer/wikia-inc./id422467077',
 			'google-play' => 'https://play.google.com/store/apps/developer?id=Fandom+powered+by+Wikia',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "google-play" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)